README.md

```diff
@@ -157,6 +160,11 @@ For getting started with that, you can look at the `extractActions()` function i
 npm install ytdl-core
 ```
 
+Or for Yarn users:
+```bash
+yarn add ytdl-core
+```
+
```

Comment:
I think instead of adding/changing this, we should just remove it outright. Like with TypeScript, surely anyone using Yarn would know how to add a node module.